README.md

```diff
@@ -225,6 +225,18 @@ Log types are exposed as `consola.[type](...)` and each is a preset of styles an
 
 A list of all available built-in types is [available here](./src/constants.ts).
 
+## Raw output
+
+Objects sent to reporter by logging them could be an issue when using conflicting key (esp. `args`).
+To enforce the object to be fully threaded with conflict, you can use the `raw` method chained to any log type.
```

Comment:
```suggestion
To enforce the object to be fully shown, you can use the `raw` method chained to any log type.
```

Reply:
woups, I made a mistake, I did want to say "treated without conflict".